lua/lvim/bootstrap.lua

```diff
@@ -63,14 +50,9 @@ function M:init(base_dir)
   self.pack_dir = join_paths(self.runtime_dir, "site", "pack")
   self.lazy_install_dir = join_paths(self.pack_dir, "lazy", "opt", "lazy.nvim")
 
-  ---@meta overridden to use LUNARVIM_CACHE_DIR instead, since a lot of plugins call this function internally
-  ---NOTE: changes to "data" are currently unstable, see #2507
-  ---@diagnostic disable-next-line: duplicate-set-field
-  vim.fn.stdpath = function(what)
-    if what == "cache" then
-      return _G.get_cache_dir()
-    end
-    return vim.call("stdpath", what)
+  -- for older installs. NVIM_APPNAME defaults to "lvim"
+  if not vim.env.NVIM_APPNAME then
+    vim.env.NVIM_APPNAME = "lvim"
```

Comment:
does it work if it's under the lines above? it looks like they get the directories before APPNAME would be set so self.runtime_dir, self.config_dir and self.cache_dir would point to nvim dirs